### PR TITLE
2nd Patreon Characters Crusade

### LINKED ISSUE #1353: Second Patreon Characters Crusade
ℹ️ **Development Notice**: Please note that this branch is based on [1-8-1-update](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/tree/1-8-1-update), which means it **_should not_** be merged into `master` until after the `1-8-1-update` branch has been merged.

### Links:
- 📗 [Life-Binder Codex of Eonar](https://docs.google.com/spreadsheets/d/1hKQmfMrI_d8FtRez5w68OfS1s0JVr5r2yF8nryGe104)

## Changelog:
- Added **~XX** Patreon characters.

### Draenei

**Courtiers**
**Counts**
**Barons**

### Dwarves

**Counts**

**Barons**

**Courtiers**

### Gnolls

**Courtiers**

### Goblins

**Counts**

### Highborne

**Courtiers**

### High Elves

**Counts**

**Courtiers**

### Humans

**Counts**

**Courtiers**

### Liches

**Counts**

**Courtiers**

### Murlocs

**Courtiers**

### Night Elves

**Counts**

**Courtiers**

### Orcs

**Courtiers**

### Tauren

**Courtiers**

### Vrykul

**Counts**

## To-do list:
- [ ] Give lands to characters.
- [ ] Balance the traits of characters. Customization points must not exceed achievement limit.
- [x] Check other characters for balance issues and note this in [the spreadsheet](https://docs.google.com/spreadsheets/d/1hKQmfMrI_d8FtRez5w68OfS1s0JVr5r2yF8nryGe104).

### PR DESCRIPTION
**Pull request by hamjustine**
### Links:
- 📗 [Life-Binder Codex of Eonar](https://docs.google.com/spreadsheets/d/1hKQmfMrI_d8FtRez5w68OfS1s0JVr5r2yF8nryGe104)

## Diones Changelog:
- Added potential classes to **~50** Patreon characters in anticipation for 2025 magic update
- Added **~20** Patreon characters. Including DNA, History, Titles and Provinces
- Edited **~3** Existing Patreon characters with new updates. Including DNA, History, Titles and Provinces

## Herbe's Changelog:
- Added **~2** Patreon characters.

## Skesus's Changelog:
Added **~13** Patreon characters. Including DNA, History, Titles and Provinces
Edited **~3** Existing Patreon characters with new updates. Including DNA, History, Titles and Provinces

### Blood Elves / High Elves
- Kalhas Aurion
- Mathemarion Mognaur
- Zagryzoń Gryzwrath
- Alaei Silversun
- Morde'Kain Bloodsinger
- Atia Silverswell
- Merrillith Miresong 
- Allra Freenrunner 

### Humans
- Athael Gresham 
- Harold Konifitz 
- Tarian Pendraig 
- Dazireh Sillendis 
- Lorsan Wildthorn (and a mother, father, and sister for him)
- Iordanos Agios
- Zula Felborn
- Yvonne Angliofin
- Caldarius Blackwood
- Sophia Westhall
- Krenath Cloudmount
- Nicholas Quintillus

### Gnolls
- Tropaion Slabmaw 

### Orcs
- Durnack Hellfire
- Khazid Firestorm 

### Vulpera
- Kayn Valargent 

### Dark Iron Dwarf
- Zake Zakehan 

### Night Elf
- Ash'al Marrowsong 
- Dione Forestsong
- Alu Stillblade
- Kedre Monarchwing and his family

### Goblins
- Harvestor Treefell

### Highborne
- Nova Starleader

### Tauren
- Raizen Bluff

### Naga
- Vaj'esha N'razji 

### Draenei 
- Akasta Toli

### CODE REVIEW

**Comment by Spectral-Baron** [comment]
went through placement of characters/traits/age of all the characters listed.
Every character I don't mention here seemed just fine, nothing that I noticed being terribly inconsistent with excel sheet.

Here are a few things you may want to consider;
- Tarian Pendraig has no education or personality traits in 583, making him fairly weak stat wise - game did not generate him more
- Durnack Hellfire main age is listed as 18, but char is 27 at main bookmark. He's also missing Herculean.
- Zake Zakehan starts as Ragnaric, but becomes Al'Akiric for 603 and 605 - should be Ragnaric or illidari according to excel
- Cookie Cookie doesn't have herbalist trait

File: history/characters/34000_murloc.txt [hunk 1]
```diff
@@ -17559,19 +17559,6 @@ murloc143={
...
-	culture=murloc religion=water_deities
-	trait=education_martial_3 trait=brave trait=patient trait=greedy 
-	583.4.10={ birth=yes trait=creature_murloc }
-	598.3.17={
-		employer=5811 #Edwin VanCleef
-		religion=masonry
-	}
-	655.9.22={ death=yes }
-}
-
```
Comment (Grekopithikos) [inline history/characters/34000_murloc.txt:17574]: Why did you remove Cookie?

Comment (skesus) [reply history/characters/34000_murloc.txt:17574]: 1.  Tarian becomes Martial 4 in a later bookmark when he becomes a Paladin. Feel free to give it to him sooner, I felt his backstory meant he gained martial abilities later in life after becoming a Paladin. But idc.
2. Durnack is listed twice, the more recent bookmark he is 18, not 27. I'm assuming the patreon updated what he wanted for his character in the more recent entry. Also he is supposed to have Herculean in the most recent update and no longer has Genius.
3. I don't know why he adopts Al'akiric, but I gave him Illidari at 603.1.1
4. I made a mistake and forgot to include "lifestyle_" in front of herbalist.

**Comment by helloruhe** [comment]
can u guys add my charactref

File: history/characters/34000_murloc.txt (hunk 1 above)
Comment (Grekopithikos) [reply history/characters/34000_murloc.txt:17574]: We should probably make a rule about making lore characters patreon characters.
Imagine someone asking to add Uther as a patreon.
I know their significance differs but the point still stands.

**Comment by hamjustine** [comment]
> can u guys add my charactref

maybe, how much do you have on you? :p

File: history/characters/34000_murloc.txt (hunk 1 above)
Comment (hamjustine) [reply history/characters/34000_murloc.txt:17574]: I added the `lifestyle_` in front of `herbalist` for cookie. But we have to decide what to do with this character, maybe ask the patreon and let him for now ?

Comment (Spectral-Baron) [reply history/characters/34000_murloc.txt:17574]: > 1. Tarian becomes Martial 4 in a later bookmark when he becomes a Paladin. Feel free to give it to him sooner, I felt his backstory meant he gained martial abilities later in life after becoming a Paladin. But idc.

that's fine, but iirc you don't get any events to get an education if you don't already have one after becoming an adult. nbd as he's a courtier, but might be better to start him off at Martial 1 or 2 if backstory progression is your intent.

> 2. Durnack is listed twice, the more recent patreon update he is 18, not 27. I'm assuming the patreon updated what he wanted for his character in the more recent entry. Also he is supposed to have Herculean in the most recent update and no longer has Genius.

seems he has the strong trait instead of Herculean, checked all start dates.

> 3. I don't know why he adopts Al'akiric, but I gave him Illidari at 603.1.1

Yeah real bizzare. Paused game on start date, still Al'Akiric for me in 605 and 603

> 4. I made a mistake and forgot to include "lifestyle_" in front of herbalist.

looks good

**Comment by hamjustine** [comment]
after discussion, we removed the patreon cookie and put back the old one.

File: history/titles/00_k_quelthalas.txt [hunk 2]
```diff
@@ -37,11 +37,7 @@ c_shylfe_anore = {
 		liege=e_quelthalas
 	}
 	10.1.1={holder = 42002}
-	603.1.30={holder=42003}
-	604.1.1={
-		holder = 42075
-		liege = k_quelthalas
-	}
+	583.1.1={holder=patron310657} # Mathemarion Mognaur patreon
```
Comment (Grekopithikos) [inline history/titles/00_k_quelthalas.txt:40]: Shouldn't the patreon be in only one in one bookmark?
If it's like that then he'll be in all 3

File: history/titles/00_k_quelthalas.txt [hunk 3]
```diff
@@ -50,14 +46,8 @@ c_falconwing_square = {
...
 
-	603.1.30={holder=42003}
-
-	603.8.1={
-		holder = 42078
-		liege = k_tranquillien
-	}
+	583.1.1={holder=patron310655} # Kalhas Aurion patreon	
 	605.4.14={
-		holder = 42075
```
Comment (Grekopithikos) [inline history/titles/00_k_quelthalas.txt:60]: same thing

File: history/titles/00_k_quelthalas.txt (hunk 2 above)
Comment (hamjustine) [reply history/titles/00_k_quelthalas.txt]: changed in c725d75db7866fb47146352ee930afd1c408fc65

File: history/titles/00_k_quelthalas.txt (hunk 3 above)
Comment (hamjustine) [reply history/titles/00_k_quelthalas.txt:60]: changed in c725d75db7866fb47146352ee930afd1c408fc65

File: history/titles/00_k_quelthalas.txt [hunk 4]
```diff
@@ -37,11 +37,8 @@ c_shylfe_anore = {
 		liege=e_quelthalas
 	}
 	10.1.1={holder = 42002}
-	603.1.30={holder=42003}
-	604.1.1={
-		holder = 42075
-		liege = k_quelthalas
-	}
+	583.1.1={holder=patron310657} # Mathemarion Mognaur patreon
+	605.4.14=={holder = 42002}
```
[the remark this answers is not in the document]
Comment (hamjustine) [reply history/titles/00_k_quelthalas.txt]: fixed !

File: history/titles/00_k_quelthalas.txt (hunk 3 above)
Comment (Grekopithikos) [reply history/titles/00_k_quelthalas.txt:60]:
```
583.1.1={holder=patron310655} # Kalhas Aurion patreon
603.1.30={holder=42003}

603.8.1={
	holder = 42078
	liege = k_tranquillien
}
605.4.14={
	holder = 42075
}
```

**Comment by Raged50** [comment]
> 09:38:49][characterhistory.cpp:761]: Missing loc for name 'Ash'al' for character 'patron310663'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Allra' for character 'patron310660'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Kalhas' for character 'patron310655'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Merrillith' for character 'patron310658'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Dione' for character 'patron310669'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Zake' for character 'patron310664'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Tarian' for character 'patron310666'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Tropaion' for character 'patron310661'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Mathemarion' for character 'patron310657'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Khazid' for character 'patron310665'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Dazireh' for character 'patron310667'
> [09:38:49][characterhistory.cpp:761]: Missing loc for name 'Maelouen' for character 'patron310656'
> [09:38:50][characterhistory.cpp:488]: 340000 is not landed and cannot be set as employer of patron310663 in 572.4.25 the command won't execute.
> [09:38:50][characterhistory.cpp:866]: Historical child character with ID patron310667 is missing a childhood trait
